.travis.yml

```diff
@@ -0,0 +1,21 @@
+# Only build and run local tests (no UI tests are executed)
+
+language: android
+android:
+  components:
+    - tools
+    - platform-tools
+    - build-tools-25.0.0
+    - android-25
+    - extra-android-m2repository
```

Comment:
For the new release we need to switch to GMaven. We will not publish to SDK Manager anymore.